dev-libs/freexl/freexl-1.0.5.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="Simple XLS data extraction library"
+HOMEPAGE="https://www.gaia-gis.it/fossil/freexl/index"
+SRC_URI="http://www.gaia-gis.it/gaia-sins/${PN}-sources/${P}.tar.gz"
+
+LICENSE="MPL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~x86"
+IUSE="static-libs"
```

Comment:
Please disable static libs unconditionally, they are not needed by any packages and are a security nightmare.

Reply:
Done. Note configuring with `--disable-static` still produces an `.la` file so the deletion in `src_install` is still necessary.